09-hypothesis-testing.Rmd

```diff
@@ -85,13 +85,19 @@ Let's start with an activity studying the effect of gender on promotions at a ba
 
 Say you are working at a bank in the 1970s and you are submitting your résumé to apply for a promotion. Will your gender affect your chances of getting promoted? To answer this question, we'll focus on data from a study published in the _Journal of Applied Psychology_ in 1974. This data is also used in the [*OpenIntro*](https://www.openintro.org/) series of statistics textbooks. 
 
-To begin the study, 48 bank supervisors were asked to assume the role of a hypothetical director of a bank with multiple branches. Every one of the bank supervisors was given a résumé and asked whether or not the candidate on the résumé was fit to be promoted to a new position in one of their branches. 
+```{r echo=FALSE, purl=FALSE}
+# This code is used for dynamic non-static in-line text output purposes
+n_resumes <- promotions %>% nrow()
+n_resumes_gender <- n_resumes/2
```

Comment:
@rudeboybert Let's make sure to run the {styler} all the way through again after this refactoring too just to make sure we are consistent throughout. Not sure if it will catch inline R code too though.

Reply:
Added to #349 updated release steps